tests/unit/windows/TestUwbAppConfiguration.cxx

```diff
@@ -114,8 +133,67 @@ TEST_CASE("UwbAppConfiguration performs allocation for contained value correctly
         };
         UwbAppConfigurationParameter appConfiguration{ dstMacAddressExpected, UWB_APP_CONFIG_PARAM_TYPE_DST_MAC_ADDRESS };
 
-        auto& appConfigurationDdi = appConfiguration.DdiParameter();
-        auto& dstMacAddressActual = appConfiguration.Value();
+        const auto& appConfigurationDdi = appConfiguration.DdiParameter();
+        const auto& dstMacAddressActual = appConfiguration.Value();
         REQUIRE(dstMacAddressActual == dstMacAddressExpected);
     }
 }
+
+TEST_CASE("UwbSetAppConfigurationParameters performs allocation for contained values correctly", "[basic]")
```

Comment:
You might already be working on it, but we'll definitely want an explicit test case for each of the unique types being converted: enum, integral (if any), `RangingMethod`, `std::unordered_set<::uwb::protocol::fira::ResultReportConfiguration>>`, `UwbMacAddress` with both lengths (short [2], extended [8]). Those types come from the `std::visit` function in `GenerateUwbSetAppConfigParameterDdi`.